packages/providers/onekey-neo-provider/README.md

```
@@ -0,0 +1 @@
+# OneKey Neo Provider 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)